warrior/Tools/tc_generator/templates/netconf_test_data_template.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+<configs>
+
+  <config_data name='addUserConfig'>
+     <config>
+      <users xmlns="urn:fujitsu:params:xml:ns:yang:usersecurity">
+      <user>
+        <username>warrioruser</username>
+        <group>level-4</group>
+        <adminState>allow</adminState>
+      <password>JUnk123</password>
+      </user>
+    <user>
+        <username>sample</username>
+        <group>level-4</group>
+        <adminState>allow</adminState>
+      <password>JUnk123</password>
```

Comment:
remove the default password.

Reply:
Made necessary changes as you suggested .